assets/encrypted_pan_list.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@alberto-re-pagopa @TommasoLencioni @GiovanniMancini dear, could you tell me if this file put in this folder is ok or if it would be better to put it in another place? Thanks.

Reply:
to me looks like the right place